### PR TITLE
Make the target installation directory configurable via DTC_ROOT

### PR DESCRIPTION
**Pull request by obfischer**
Changed environment variable `DTC_ROOT` to be configurable specify the parent directory the installation directory, which defaults the value of the environment variable `HOME`.

### All Submissions:

* [x] Did you update the `changelog.adoc`?
* [ ] Does your PR affect the documentation?
* [ ] If yes, did you update the documentation or create an issue for updating it?

The source of the documentation can be found in `/src/docs/`.

If you didn't find the time to update docs, please create an issue as reminder to do so.

### Your first submission

* [ ] Welcome to the list of contributors! If you have any questions, feel free to ask them by creating a new issue
* [x] Have you added your name to the list of [30_community.adoc](https://github.com/docToolchain/docToolchain/blob/ng/src/docs/10_about/30_community.adoc)?

inspiration: https://github.com/stevemao/github-issue-templates

### CODE REVIEW

File: changelog.adoc [hunk 1]
```diff
@@ -54,6 +54,7 @@ and this project tries to adhere to https://semver.org/spec/v2.0.0.html[Semantic
 ** 'org.openapi.generator:6.6.0'
 ** 'org.spockframework:spock-core:2.3-groovy-3.0'
 * `dtcw` and `dtcw.ps1`:
+** Changed environment variable `DTC_ROOT` to be configurable specify the parent directory the installation directory, which defaults the value of the environment variable `HOME`
```
Comment (mh182) [inline changelog.adoc:322]: I introduced `DTC_ROOT` as prerequisite to address #1062. Since `DTC_ROOT` was not accessible until now, I would change the description a bit. What about:

"Add new environment variable `DTC_ROOT` to make the docToolchain installation directory configurable. The default installation directory remains $`HOME/.docoolchain`.

File: src/docs/010_manual/20_install.adoc [hunk 2]
```diff
@@ -127,14 +127,18 @@ container engine and pull the docToolchain image on the first invocation.
 [role='primary']
 --
 
-To install docToolchain in `$HOME/.doctoolchain` execute the following command.
+docToolchain will be installed below the directory specified via the environment variable `DTC_ROOT`, which defaults to `$HOME`.
+To install docToolchain in `$DTC_ROOT/.doctoolchain` execute the following command.
```
Comment (mh182) [inline src/docs/010_manual/20_install.adoc:191]: This is erroneous - check your code above. `DTC_ROOT=${HOME}/.doctoolchain`. I would rephrase the section above with something like:

"Using the wrapper script will install docToolchain into `$HOME/.doctoolchain`."

Your tip already provides all the information a user may need to change docToolchain's installation directory. So why explaining it redundantly.

File: src/docs/010_manual/20_install.adoc [hunk 3]
```diff
@@ -127,14 +127,18 @@ container engine and pull the docToolchain image on the first invocation.
...
+TIP: You can specify an alternative installation directory by setting `DTC_ROOT` to a custom value.
+Please keep in mind the `DTC_ROOT` must be permanently specified, otherwise `dtcw` will not be able to find your installation.
 
 [source, bash]
 ----
 ./dtcw install doctoolchain
 ----
 
-In case you have no Java installed you may use `dtcw` to install Java in a sub-directory of `$HOME/.doctoolchain`.
+In case you have no Java installed you may use `dtcw` to install Java in a sub-directory of `$DTC_ROOT/.doctoolchain`.
```
Comment (mh182) [inline src/docs/010_manual/20_install.adoc:201]: Once again, `DTC_ROOT` points to `$HOME/.doctoolchain`. According to your documentation Java would be installed in `$HOME/.doctoolchain/.doctoolchain`.

File: src/docs/010_manual/20_install.adoc (hunk 2 above)
Comment (mh182) [reply src/docs/010_manual/20_install.adoc:191]: We should move this paragraph just after the header. The Windows wrapper script `dtcw.ps1` has to be changed as well.

**Review by mh182 — CHANGES_REQUESTED** [review]
I would like to have test case(s) for the correct installation of docToolchain and Java when `DTC_ROOT` is provided as an alternative.

Look at the `test/README.md` how to add new tests. If you need help, drop me a line.

**Comment by obfischer** [comment]
I will close this pull request, as it is outdated.
